src/System.Management.Automation/FormatAndOutput/common/BaseOutputtingCommand.cs

```diff
@@ -743,6 +743,27 @@ private void ProcessCachedGroupOnWide(WideViewHeaderInfo wvhi, List<PacketInfoDa
             _formattingHint = hint;
         }
 
+        static private int TryGetColumnsOnTheScreen(int columnNumber)
```

Comment:
Usually `Try...()` fuctions return `bool`. 
Also we get mainly `Console.WindowWidth`. So I think we could use `GetConsoleWindowWidth` name.

Reply:
That's right. I will rename `TryGetColumnsOnTheScreen` to `GetConsoleWindowWidth`.